server/controllers/getRequests.js

```diff
@@ -1,6 +1,8 @@
 var mongoose = require('mongoose');
 var Booking = require('../../database/index.js').Booking
+var moment = require('moment');
 mongoose.connect('mongodb://localhost/reservations', {useNewUrlParser: true});
+var prevMonths = [1,2,3,4,5,6,7,8,9,0,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31]
```

Comment:
Just out of curiosity, why do you list previous months up to 31 - is that an arbitrary limit or for a specific reason?

Reply:
That is an idea I was playing with for when the client asks for a calendar for a month that has already passed. I need to change the state to say that every day is reserved, which translates on the client side to "no available dates". I will be scraping this piece and figuring out a more dynamic way to achieve this goal. Good eye!